libminifi/include/utils/file/FileUtils.h

```diff
@@ -25,14 +25,7 @@
 #include <utility>
 #include <vector>
 
-#ifdef USE_BOOST
-#include <dirent.h>
-#include <boost/filesystem.hpp>
-#include <boost/system/error_code.hpp>
-#ifndef WIN32
-#include <sys/stat.h>
-#endif
-#else
+#ifndef USE_BOOST
```

Comment:
We can now remove this check and just assume that it's not defined. We will eventually get rid of all boost code from libminifi.

Reply:
I have tried removing it but that failed windows build. I will try to optimize imports by moving relevant includes to ifdef WIN32 scope